Makefile

```diff
@@ -1 +1,5 @@
+lab5: main.cpp coordinates.cpp coordinates.h
+	g++ -Wall -Werror -lgtest -std=c++11 main.cpp coordinates.cpp coordinates.h -o lab5
```

Comment:
You can omit the header files from compile commands. They become part of the .cpp file with `#include`.